Updates to setup.py and top-level readme file

namurphy opened the pull request:
This pull request includes:
- Updating `setup.py`, including giving us a version number of `'0.0.1.dev1'` and adding some keywords
- Updating the top-level readme with respect to the project status and linking to the how to contribute guide
- Deleting a temporary script that I was using prior to Travis CI

Ideas for a versioning scheme are discussed more in issue https://github.com/PlasmaPy/PlasmaPy/issues/25.  The version number above is therefore tentative and could change.  We will still need to edit `setup.py` when set up a contact email address for the project and when we have a webpage other than the GitHub repository.

Some additional questions:
- Would anyone like to be added to the list of people to contact if you have questions?
- Are there any additional keywords we should add beyond 'plasma', 'plasma physics', and 'science'?

SolarDrew commented on README.md:
```diff
@@ -4,22 +4,22 @@ A community developed Python package for plasma physics.
 
 ## Installation
 
 In the top level directory, run
```
This should first say how to clone the top level directory from GitHub

pep8speaks commented:
Hello @namurphy! Thanks for updating the PR.

 - In the file [`plasmapy/__init__.py`](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/plasmapy/__init__.py), following are the PEP8 issues :

> [Line 10:1](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/plasmapy/__init__.py#L10): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 11:1](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/plasmapy/__init__.py#L11): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 12:1](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/plasmapy/__init__.py#L12): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 13:1](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/plasmapy/__init__.py#L13): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 16:1](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/plasmapy/__init__.py#L16): [W391](https://duckduckgo.com/?q=pep8%20W391) blank line at end of file

 - In the file [`setup.py`](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/setup.py), following are the PEP8 issues :

> [Line 6:44](https://github.com/PlasmaPy/PlasmaPy/blob/4f0a8b91f1b16de45bab5e54e2fc7573e6ef6226/setup.py#L6): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace

namurphy commented:
My recent changes:
- Added @SolarDrew as a contact
- Added instructions on how to download/clone repository
- Removed underscores in numbers in a test as it was a Python 3.5 incompatibility that only slightly improved readability
- Added a mention of the PEP8 Speaks GitHub integration that @SolarDrew just added and is working (thank you!)

In an attempt to be pure evil, I am for the moment leaving a line that is one character too long since changing it would make it less readable.

For the most part, this pull request is a lot of little changes.
